internal/jobmanager/manager.go

```diff
@@ -0,0 +1,86 @@
+package jobmanager
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/aadc-dev/teleport-job-management-int/lib/jobworker"
+
+	"github.com/google/uuid"
+)
+
+// Manager handles Jobs
+type Manager struct {
+	mu   sync.RWMutex
+	jobs map[string]*jobworker.Job
+}
+
+// MakeManager makes a new instance of job manager
+func MakeManager() *Manager {
+	return &Manager{
+		jobs: make(map[string]*jobworker.Job),
+	}
+}
+
+// CreateAndStartJob creates a new job and starts it.
+// owner is the CN from the client certificate.
+func (m *Manager) CreateAndStartJob(ctx context.Context, command string, args []string, owner string) (*jobworker.Job, error) {
+	// TODO: add more robust guard clauses and return named errors to map to correct httpStatusCodes for our server calls.
+	if len(command) == 0 {
+		log.Printf("Validation error: command cannot be empty for owner '%s'", owner)
+		return nil, fmt.Errorf("command cannot be empty")
```

Comment:
Tip: if you have only a static error message, `errors.New` is much more efficient than `fmt.Errorf`.
```suggestion
		return nil, errors.New("command cannot be empty")
```

Reply:
awesome! Thanks!